documentation/adr/0009-use-compiled-matlab-for-python.md

```diff
@@ -0,0 +1,44 @@
+[<-previous](0008-pace-syntax-guiding-principles.md) | next->
+
+# 9. Use Compiled Matlab library for Python users to run PACE
+
+Date: 2020-Nov-29
+
+## Status
+
+Accepted
+
+
+## Context
+
+PACE is a collection of programs written in various languages including Matlab
+and Python, but the aim is that *users* should be able to use either Python
+*or* Matlab to run PACE programs.
+
+Matlab has [built-in functionality](https://www.mathworks.com/help/matlab/call-python-libraries.html)
+to call Python programs however the reverse is not true - principally because
+Matlab requires a (paid-for) license to use.
+
+Thus to allow Python users to call the current Matlab PACE codes
+([Horace](https://github.com/pace-neutrons/Horace/)/[Herbert](https://github.com/pace-neutrons/Herbert/))
+it should either be translated to Python or C++ and a wrapper to Python and Matlab created,
+or the Matlab Compiler Runtime (MCR) toolbox should be used to "compile" 
+`horace`/`herbert` for distribution as a Python package, as detailed
+[here](../../python_interface/design/01_pace_python_high_level_discussion.md).
```

Comment:
It should mention somewhere that you can run compiled Matlab without a license. It's implied and is in the linked documents, but would probably help if it was mentioned explicitly.